importScripts/sql/testImports/subjectList.sql

```diff
@@ -16,13 +16,14 @@ UNION(SELECT s_SubjectID FROM SubjectsExport WHERE os_DateMax IS NOT NULL LIMIT
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE o_Accession_Full_ID IS NOT NULL LIMIT 2)
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE o_ItemType IS NOT NULL LIMIT 2)
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_SubjectSubjectID IS NOT NULL LIMIT 2)
-UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_SubjectID IS NOT NULL LIMIT 2)
-UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_SubjectName IS NOT NULL LIMIT 2)
+UNION(SELECT s_SubjectID FROM SubjectsExport WHERE relatedSubject_SubjectID IS NOT NULL LIMIT 2)
+UNION(SELECT s_SubjectID FROM SubjectsExport WHERE relatedSubject_SubjectName IS NOT NULL LIMIT 2)
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_Association IS NOT NULL LIMIT 2)
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_Dates IS NOT NULL LIMIT 2)
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_DateMin IS NOT NULL LIMIT 2)
 UNION(SELECT s_SubjectID FROM SubjectsExport WHERE ss_DateMax IS NOT NULL LIMIT 2)
-UNION(SELECT s_SubjectID FROM SubjectsExport WHERE rs_SubjectType IS NOT NULL LIMIT 2)
+UNION(SELECT s_SubjectID FROM SubjectsExport WHERE relatedSubject_SubjectType IS NOT NULL LIMIT 2)
+UNION(SELECT s_SubjectID FROM SubjectsExport WHERE s_SubjectName LIKE 'PRINSEP%' OR relatedSubject_SubjectName LIKE 'PRINSEP%' OR relatedSubject_SubjectName LIKE 'Bussel%' OR s_SubjectName LIKE 'Bussel%')
```

Comment:
Extra sample records to match the problem names: https://youtrack.gaiaresources.com.au/youtrack/issue/RWAHS-147